source/thememodifier.yaml

```diff
@@ -1,3 +1,6 @@
 Constants:
     - DetailsViewDescriptionWidth(150,750): 'Details View: Width of description'
-    - DetailsViewDescriptionDockToRight: 'Details View: Dock description to the right'
\ No newline at end of file
+    - DetailsViewDescriptionDockToRight: 'Details View: Dock description to the right'
+    - DetailsViewBackgroundVerticalAlignment: 'Details View: Vertical alignment of the background image.'
+    - DetailsViewBackgroundHorizontalAlignment: 'Details View: Horizontal alignment of the background image.'
+    - DetailsViewLogoVisibility: 'Details View: Logo visibility.'
```

Comment:
Change the description please to reflect that this change is different to the setting in ExtraMetadataLoader extension. Maybe something like `Details View: Force Logo visibility (Overrides plugin settings)` or something similar you can think of.